scripts/g.extension/g.extension.py

```diff
@@ -2277,7 +2277,9 @@ def main():
 
     if options['operation'] == 'add':
         check_dirs()
-        get_addons_paths(gg_addons_base_dir=options['prefix'])
+        # check if extension is from official github
+        if original_url == '':
```

Comment:
Yes, the condition is correct. Plus we need handle HTTPException inside `download_addons_paths_file()` function.